Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -306,7 +306,8 @@ static void logProcessPoolState(const WebProcessPool& pool)
     });
 
 #if USE(EXTENSIONKIT)
-    AuxiliaryProcessProxy::setManageProcessesAsExtensions(CFPreferencesGetAppBooleanValue(CFSTR("manageProcessesAsExtensions"), CFSTR("com.apple.WebKit"), nullptr));
+    bool manageProcessesAsExtensions = !CFPreferencesGetAppBooleanValue(CFSTR("disableProcessesAsExtensions"), kCFPreferencesCurrentApplication, nullptr);
```

Comment:
That is a good point! In this case, I believe we do not have the normal WebKit feature flags settings available yet, since those are related to a page, which has not been created yet. This is a temporary check, that will be removed fairly soon.

Thanks for reviewing!

Reply:
Ok, I'll approve based on the understanding that this is a temporary check.